stdlib/REPL/src/REPL.jl

```diff
@@ -1168,17 +1168,17 @@ function run_frontend(repl::StreamREPL, backend::REPLBackendRef)
     dopushdisplay = !in(d,Base.Multimedia.displays)
     dopushdisplay && pushdisplay(d)
     while !eof(repl.stream)
-        if have_color
+        if have_color === true
```

Comment:
Instead I would set `have_color = get(repl.stream, :color, false)` at the top of this function.

Reply:
Right